packages/design-system/src/styles/base/_link.scss

```diff
@@ -1,17 +1,41 @@
 /* stylelint-disable selector-class-pattern */
 @import '../settings/index.scss';
 
+// Links
+// Default link
+$link-color: $color-primary !default;
+$link-visited-color: $color-visited !default;
+$link-hover-color: $color-primary-darker !default;
+$link-active-color: $color-primary-darkest !default;
+
+// Inverse link
+$link-inverse-color: $color-base-inverse !default;
+$link-inverse-visited-color: $color-muted-inverse !default;
+$link-inverse-hover-color: $color-muted-inverse !default;
+$link-inverse-active-color: $color-muted-inverse !default;
+
+// Link underline
+$link-underline-height: 1px !default;
+$link-underline-height-hover: 1px !default;
+$link-underline-offset: 0 !default;
+
 %link {
-  color: $color-primary;
+  color: $link-color;
   cursor: pointer;
   text-decoration: underline;
+  text-decoration-color: $link-color;
+  text-decoration-thickness: $link-underline-height;
```

Comment:
For my own education, how does the design system handle CSS properties that are not supported in legacy browsers. For example `text-decoration-thickness` [doesn't look to be supported in IE11](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness#browser_compatibility). Is that okay?

Like for consuming applications, do they know there might be some stylistic differences in IE for things not supported?

Reply:
This is a great question, we aim for graceful degradation and try to strike a balance between using modern practices where applicable while ensuring the design doesn't break. 

As far as what applications teams know or don't know - I'm unsure on that.